### PR TITLE
Implementation

### PR DESCRIPTION
**Pull request by badrAZ**

### CODE REVIEW

**Review by olivierlambert — APPROVED** [review]
LGTM

File: src/index.js [hunk 1]
```diff
@@ -105,6 +106,10 @@ class BackupReportsXoPlugin {
         `  - End time: ${String(end)}`,
         `  - Duration: ${duration}`
       ].join('\n'))
+
+      nagiosText.push([
+        call.error ? `[ ${vm ? vm.name_label : 'undefined'} : ${call.error.message} ]` : ''
+      ])
```
Comment (julien-f) [inline src/index.js]: Do not push an empty entry.

**Review by julien-f — CHANGES_REQUESTED** [review]

File: src/index.js [hunk 2]
```diff
@@ -107,9 +107,11 @@ class BackupReportsXoPlugin {
         `  - Duration: ${duration}`
       ].join('\n'))
 
-      nagiosText.push([
-        call.error ? `[ ${vm ? vm.name_label : 'undefined'} : ${call.error.message} ]` : ''
-      ])
+      if (call.error) {
+        nagiosText.push([
+          `[ ${vm ? vm.name_label : 'undefined'} : ${call.error.message} ]`
+        ])
```
Comment (julien-f) [inline src/index.js]: Why are you pushing an array?

**Review by julien-f — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 2 above)
Comment (badrAZ) [reply src/index.js]: I push  each VM who has a call error.

**Review by julien-f — APPROVED** [review]

**Pull request merged**